src/map.ts

```diff
@@ -4,6 +4,7 @@
 /* We do not want to have to include DOM types just for this check */
 declare const window: unknown;
 declare const self: unknown;
+declare const global: unknown;
```

Comment:
this also looks like its added without a usage?

Reply:
It's used in the `getGlobal` function, I'm worried GH is messing something up.